lib/Doctrine/Common/Proxy/ProxyGenerator.php

```diff
@@ -480,11 +493,21 @@ public function {$returnReference}__get(\$name)
      * @param \Doctrine\Common\Persistence\Mapping\ClassMetadata $class
      *
      * @return string
+     * @throws \ReflectionException
```

Comment:
Please remove, besides being private API only, this can never happen as we check for `hasMethod()` before calling `getMethod()`.